docs/orchestrators/README.md

```diff
@@ -31,3 +31,101 @@ There may be cases where only a subset of rows is desired to be extracted in the
 process. Here the orchestrator offers the method `.filter_with` which allows 
 additional transformation steps to be injected before the rows are finally appended 
 to the delta table.
+
+# Eventhub to medallion architecture
+
+*"A medallion architecture is a data design pattern used to logically organize data in a lakehouse, with the goal of incrementally and progressively improving the structure and quality of data as it flows through each layer of the architecture:* 
+
+*(from Bronze ⇒ Silver ⇒ Gold layer tables).*
+
+*Medallion architectures are sometimes also referred to as "multi-hop" architectures."*
+
+[Source: Databricks.com](https://www.databricks.com/glossary/medallion-architecture)
+
+## EventHub to Bronze 
+
+The class `EhToDeltaBronzeOrchestrator` has been designed to carry out the task of ingest eventhub data to a bronze layer. Data is always _appended_ to the bronze table. By utilizing `EhJsonToDeltaExtractor` from [previous section](#-EventHub-to-Delta) data is extracting incrementally.
+
+*"The Bronze layer is where we land all the data from external source systems. The table structures in this layer correspond to the source system table structures "as-is," along with any additional metadata columns that capture the load date/time, process ID, etc. The focus in this layer is quick Change Data Capture and the ability to provide an historical archive of source (cold storage), data lineage, auditability, reprocessing if needed without rereading the data from the source system."*
+
+[Source: Databricks.com](https://www.databricks.com/glossary/medallion-architecture)
+
+Using the orchestrator without adding any filtering `.filter_with` the output schema is the following:
+
+| **Column Name**   | **Data type** | **Explanation**                                                                                                    |
+|-------------------|---------------|--------------------------------------------------------------------------------------------------------------------|
+| BodyId            | Long          | A ID generated to give a unique id for each row in the bronze table.  Calculated based on sha2 hashing the *Body*. |
```

Comment:
Can you please hash the Body *and* the EnueuedTimestamp. It is absolutely allowed to resend the same message multiple times. The enqueued timestamp will usually differ in those cases.

This does not even solve all use cases, because it is possible to bulk-submit to EventHub where the Body and the EnueuedTimestamp of multiple messages can both batch if they were submitted in the same bulk. But i think that it is OK to disregard this case since I cannot think of a legitimate use case where one might do that.

Reply:
Added a new hash column: "EventhubRowId"